### PR TITLE
Fix changelog

### PR DESCRIPTION
**Pull request by opotowsky**
I noticed that some spacing issues was preventing some of the lists rendering as such (and so they were paragraphs with asterisks separating entries). I have done my best to fix this formatting problem. In most cases, it is because there needs to be a minimum of 3 spaces to create a new nested list. 

Also changed: 
- Single ` are now double `` uniformly
- Lines 64 and 185 have what I believe to be corrected typos -- please check that these make sense
- Tried to create uniform blank spacing

### CODE REVIEW

**Comment by opotowsky** [comment]
Let me know if you want me to include this PR in the changelog ;-)

File: CHANGELOG.rst [hunk 1]
```diff
@@ -15,18 +15,18 @@ v0.7.0 RC2
 
 * Subvoxel R2S allows activation and photon source sampling by
   material within a mesh voxel
```
Comment (gonuke) [inline CHANGELOG.rst]: I think that if the above is on a single line, the formatting might work better.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -15,18 +15,18 @@ v0.7.0 RC2
...
 * OpenMC support
-  * Add CI test for OpenMC as an optional dependency
-  * Add options to install OpenMC Python API
-  * Functions to read OpenMC state point file to get meshtally data
+   * Add CI test for OpenMC as an optional dependency
+   * Add options to install OpenMC Python API
+   * Functions to read OpenMC state point file to get meshtally data
 
 * File scripts/activation_responses.py to get activation responses data for
   ALARA output file for better visualization. Responses include:
```
Comment (gonuke) [inline CHANGELOG.rst]: Also here with keeping these on a single line for formatting

File: CHANGELOG.rst [hunk 3]
```diff
@@ -147,80 +145,80 @@ v0.7.0 RC2
...
+   * new check won't now be triggered after a merge only on PRs
+   * utils.py: updated the download timeout time to 120sec (from 30sec)
+   * updated CI to use CircleCI 2.1 workflows: now build separately from tests with state saved between runs
+   * test_fluka:
+      * added test to check the data tag name of the different tally part and
+        error.
+   * revert internal nuc_data_path to origin value after internal data test
+   * added DEFINE variable to allow material.cpp amalgamation without decay.cpp
+   * now skips endf test when website is not reachable to allow completeness of
+     the other tests.
```
Comment (gonuke) [inline CHANGELOG.rst]: Keep the above on the same line

**Review by gonuke — CHANGES_REQUESTED** [review]
Thanks for breaking the seal on this tedious formatting tasks....
I have a few more thoughts on formatting improvements.....

File: CHANGELOG.rst (hunk 1 above)
Comment (opotowsky) [reply CHANGELOG.rst]: on it!

**Comment by opotowsky** [comment]
@gonuke from those changes, at least now all list entries with something nested under it are formatted the same (but they are the same no matter where they fall in the nesting, don't know if that's a bother or not)

**Comment by gonuke** [comment]
I think this is good.  Will need a rebase on other recent merges that affect CHANGELOG

**Comment by opotowsky** [comment]
rebased!

**Review by gonuke — APPROVED** [review]

**Comment by gonuke** [comment]
Thanks @opotowsky - I'm not waiting for tests since I know that the only file that changed is the CHANGELOG and I don't want you rebasing forever :)

**Pull request merged**